src/lazy/text/buffer.rs

```diff
@@ -2817,6 +2902,21 @@ mod tests {
         assert!(contains_escapes);
     }
 
+    #[rstest]
+    #[case::newlines("\n\r", (3,1))]
+    #[case::crlf("\r\n\r\n", (3,1))]
+    #[case::mixed("\r\n\n\r\n", (4,1))]
+    #[case::tabs("\n\t\t\t", (2,4))]
+    #[case::mix_tabs_and_newlines("\n\t\n", (3,1))]
```

Comment:
Could you please add test cases for:
* No whitespace
* Whitespace without newlines
* Whitespace that includes a comment
* Whitespace that includes a multi-line comment.